packages/rewriter/lib/threads/worker-shim.js

```diff
@@ -0,0 +1,6 @@
+if (process.env.CYPRESS_INTERNAL_ENV === 'production') {
+  throw new Error(`${__filename} should only run outside of prod`)
+}
+
+require('@packages/ts/register')
```

Comment:
look into using require('typescript') to pretranspile this and give it to workers directly

Reply:
probably good enough for now since bundling work for server is incoming